src/database/migrations/20220327150708-create-like.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable require-jsdoc */
+export async function up(queryInterface, DataTypes) {
+  await queryInterface.createTable('likes', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER
+      },
+      accomodationId: {
+        type: DataTypes.INTEGER,
+        allowNull: false
```

Comment:
you have the relationship on the models but its missing in the migrations

Reply:
I added reference in migration folder